src/content/platform/cron-triggers/index.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 10
+---
+
+# Cron Triggers
+
+## Background
+
+Cron Triggers allow users to map a cron expression to a Worker script to enable Workers to be executed on 
+a schedule. Cron Triggers are ideal for running periodic jobs for maintenance or calling third-party APIs 
+to collect up-to-date data.
+
+## Adding CRON Triggers
+
+CRON Triggers can be added to scripts with the Cloudflare API or in the dashboard on a 
```

Comment:
CRON or Cron? not sure which one is right but we should use the same casing throughout

Reply:
I do believe Cron is the correct form.

Reply:
i think so too but just wanted to double check :)

Reply:
Fixed. 👍